WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1271,10 +1273,11 @@ - (UIBarButtonItem *)saveBarButtonItem
 - (UIBarButtonItem *)shareBarButtonItem
 {
     if (!_shareBarButtonItem) {
-        WPButtonForNavigationBar *button = [self buttonForBarWithImageNamed:@"gridicons-share-ios"
-                                                                      frame:NavigationBarButtonRect
-                                                                     target:self
-                                                                   selector:@selector(sharePost)];
+        UIImage *image = [Gridicon iconOfType:GridiconTypeShareIOS];
```

Comment:
The default size for Gridicons in Gridicons-iOS is 24px, and is what you get if you don't specify a size. I intended this method (`iconOfType:`) to be used for 'I want a default-sized icon'. There's an extra parameter you can use to override the size.